runtime/src/main/cpp/ObjCExceptions.cpp

```diff
@@ -0,0 +1,125 @@
+#if KONAN_REPORT_BACKTRACE_TO_IOS_CRASH_LOG
+
+#include <dlfcn.h>
+#include <inttypes.h>
+#include <mach-o/loader.h>
+#include <CoreFoundation/CFRunLoop.h>
+#include "Natives.h"
+#include "ObjCExceptions.h"
+#include "Types.h"
+
+extern "C" OBJ_GETTER(Kotlin_Throwable_getStackTrace, KRef throwable);
+
+static void writeStackTraceToBuffer(KRef throwable, char* buffer, unsigned long bufferSize) {
+  if (bufferSize < 2) return;
```

Comment:
Why `2`?

Reply:
To make sure it doesn't overflow the buffer if no address is written.